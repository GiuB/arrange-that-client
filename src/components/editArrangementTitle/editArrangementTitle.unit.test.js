import React from 'react';
import Enzyme, {render} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json'

import EditArrangementTitle from './editArrangementTitle';

Enzyme.configure({adapter: new Adapter()});

it('EditArrangementTitle renders', () => {
    const editArrangementTitle = render(<EditArrangementTitle 
        name='First arrangement'
        handleEnter={null}
        handleEsc={null}/>);

    expect(toJson(editArrangementTitle)).toMatchSnapshot();
});