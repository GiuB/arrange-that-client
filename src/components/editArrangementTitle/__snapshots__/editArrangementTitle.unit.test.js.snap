// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditArrangementTitle renders 1`] = `
<h1
  class="MuiTypography-root-1 MuiTypography-headline-6 MuiTypography-alignLeft-14"
>
  <div
    class="MuiFormControl-root-27 MuiFormControl-marginNormal-28"
  >
    <label
      class="MuiFormLabel-root-38 MuiFormLabel-filled-42 MuiInputLabel-root-31 MuiInputLabel-formControl-32 MuiInputLabel-animated-35 MuiInputLabel-shrink-34 MuiInputLabel-outlined-37"
      data-shrink="true"
      for="outlined-name"
    >
      Arrangement Name
    </label>
    <div
      class="MuiInputBase-root-58 MuiOutlinedInput-root-45 MuiInputBase-formControl-59"
    >
      <fieldset
        aria-hidden="true"
        class="MuiNotchedOutline-root-75 MuiOutlinedInput-notchedOutline-52"
        style="padding-left:8px"
      >
        <legend
          class="MuiNotchedOutline-legend-76"
          style="width:0"
        />
      </fieldset>
      <input
        aria-invalid="false"
        autofocus=""
        class="MuiInputBase-input-68 MuiOutlinedInput-input-53"
        id="outlined-name"
        type="text"
        value="First arrangement"
      />
    </div>
  </div>
</h1>
`;
